pkg/controller/common/helpers.go

```diff
@@ -666,6 +666,7 @@ func SupportedExtensions() map[string][]string {
 	// These are RHCOS supported extensions.
 	// Each extension keeps a list of packages required to get enabled on host.
 	return map[string][]string{
+		"two-node-ha":               {"pacemaker", "pcs", "fence-agents-all"},
```

Comment:
very minor formatting nit: could you align the package names {} with the other lines?

Reply:
done.